From 9c675a1051551af26aaefc6fcd33502a51725396 Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Mon, 26 Jul 2021 17:14:51 +0000
Subject: [PATCH] IWYU: missing include for std::vector usage in devtools
 embedded message dispatcher.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix GCC build breakage:
    ../../chrome/browser/devtools/devtools_embedder_message_dispatcher.h:116:36: error: ‘vector’ in namespace ‘std’ does not name a template type
      116 |                         const std::vector<base::Value>& params) = 0;
          |                                    ^~~~~~

Bug: 819294
Change-Id: I379bf14416c3bfd3c0cdac7bb2542b79781261ec
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3041156
Reviewed-by: Joe Mason <joenotcharles@chromium.org>
Commit-Queue: José Dapena Paz <jdapena@igalia.com>
Cr-Commit-Position: refs/heads/master@{#905300}
---
 chrome/browser/devtools/devtools_embedder_message_dispatcher.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/chrome/browser/devtools/devtools_embedder_message_dispatcher.h b/chrome/browser/devtools/devtools_embedder_message_dispatcher.h
index 12f8500b7811d..4007112f39410 100644
--- a/chrome/browser/devtools/devtools_embedder_message_dispatcher.h
+++ b/chrome/browser/devtools/devtools_embedder_message_dispatcher.h
@@ -8,6 +8,7 @@
 #include <map>
 #include <memory>
 #include <string>
+#include <vector>
 
 #include "base/callback.h"
 #include "ui/gfx/geometry/insets.h"
